libs/web/src/hooks/useSwapRouter.ts

```
@@ -2,231 +1,136 @@
-          0,
-        ),
-      }))
-      .sort((a, b) => b.totalReserves - a.totalReserves)
-      .slice(0, 5)
-      .map((r) => r.route);
-  }, [routes]);
+    isLoading: routesLoading,
+    isRefetching: routesRefetching,
+  } = useRoutablePools(assetIn, assetOut, shouldFetch);
+
+  debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)